### DISCUSSION TITLE
Can client and server handleError hooks be used at the same time?

### DISCUSSION THREAD

**Discussion by akkie** [sveltejs/kit#7802] [Q&A]
Hi,

it took me some time to understand why my client `handleError` function doesn't log my error that was thrown from a load function. I thought that my server `handleError` hook acts a default error handler for all server related errors (+server files) and the client `handleError` for all client related errors (+page,+layout files).

But it seems that they are mutually exclusive. Because if I have a server `handleError` function defined, I cannot get the client `handleError` to executed, even if I throw an error in a component or layout. So the question is can they be used at the same time?

Best regards,
Christian

**Comment by teemingc** [comment]
I've tested and they seem to both work together. 
https://stackblitz.com/edit/sveltejs-kit-template-default-7kddyq?terminal=dev

What type of error was thrown is / what type of load function you are using?

**Reply by akkie** [reply]
OK, if I add `export const ssr = false;` to my `+page.ts` file, then the client side `handleError` function will be executed.
